platformio.ini

```
@@ -18,6 +18,7 @@ lib_deps =
     1655@1.0.2  ; TinyGPSPlus (has name conflict)
     6865@1.0.0  ; TM1651 Battery Display
     6306@1.0.3  ; HM3301
+    463@1.4.0 ; Adafruit TSL2591 Library
```

```suggestion
    adafruit/Adafruit TSL2591 Library@1.4.0
```

The IDs are only necessary if there are duplicate libraries on the PlatformIO registry.